Fredmanzi/EPIC-MAIL
# #164139151 built login content

#### What does this PR do?
adds login  features to the login page
#### Description of Task to be completed?
1. it helps the user to login 
2.adds login button to the login page
#### How should this be manually tested?
when the user navigates to the login page, he/should see the form to enter his credentials and login button
